content/developer/howtos/localization.rst

```diff
@@ -0,0 +1,391 @@
+
+It works with a prefix start/end, so every account where the code starts with something between start and end will have this account.group as parent group.  Furthermore, the account groups can have a parent account group as well to form the hierarchy.  
+
+
+.. example::
+  `addons/l10n_il/data/account.group.template.csv <{GITHUB_PATH}/addons/l10n_il/data/account.group.template.csv>`__.
+
+  .. csv-table::
+     :condition: odoo_dir_in_path
+     :file: {ODOO_ABSPATH}/addons/l10n_il/data/account.group.template.csv
+     :widths: 20,20,20,20,20
+     :header-rows: 1
+
+Taxes
+-----
+
+.. seealso::
+   - :ref:`Tax References <reference/account_tax>`
+   - :doc:`/applications/finance/accounting/taxation/taxes/taxes`
+
+To add taxes you first need to specify tax groups. You normally need just one tax group for every tax rate, except for the 0% as you need to often distinguish between exempt, 0%, not subject, ... taxes. 
+This model only has two required fields: name and country. Create file `data/account_tax_group_data.xml` and list the groups:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+        <data noupdate="1">
+            <record id="tax_group_tva_0" model="account.tax.group">
+                <field name="name">TVA 0%</field>
+                <field name="country_id" ref="base.ch"/>
+            </record>
+
+            ...
+        </data>
+    </odoo>
+
+.. example::
+  `addons/l10n_ch/data/account_tax_group_data.xml <{GITHUB_PATH}/addons/l10n_ch/data/account_tax_group_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/account_tax_group_data.xml
+    :condition: odoo_dir_in_path
+    :language: xml
+    :start-after: <data
+    :end-before: </data>
+
+.. example::
+  `addons/l10n_uk/data/account.tax.group.csv <{GITHUB_PATH}/addons/l10n_uk/data/account.tax.group.csv>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_uk/data/account.tax.group.csv
+    :condition: odoo_dir_in_path
+    :language: csv
+
+
+Now you can add the taxes via `data/account_tax_template_data.xml` file.  The first tax you define that is purchase/sale also becomes the default purchase/sale tax for your products.  
+
+
+.. example::
+  `addons/l10n_ae/data/account_tax_template_data.xml <{GITHUB_PATH}/addons/l10n_ae/data/account_tax_template_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ae/data/account_tax_template_data.xml
+    :condition: odoo_dir_in_path
+    :language: xml
+    :start-at: uae_sale_tax_5_dubai
+    :end-at: </record>
+
+If some accounts should use default taxes, you can set them up in `data/account_account_template_post_data.xml`
+
+Tax Report
+----------
+
+.. raw:: html
+
+   <div><span class="badge" style="background-color:#AD5E99">Enterprise feature</span><div>
```

Comment:
Might be a good use-case for an `rst_epilog` and a "standard" (project-wide) substitution.